src/api/python/pyboolector.pyx

```diff
@@ -6,6 +6,7 @@
 # This file is part of Boolector.
 # See COPYING for more information on using this software.
 #
+# cython: language_level=3
```

Comment:
This will affect Python 2 users (who shouldn’t exist, but they do).

Can’t we just invoke Cython with the correct language argument rather than doing this?

Reply:
Sure, that's fine. We get a few warnings without it set, but that's okay

Reply:
Do you mean Cython-level warnings? Like “blah blah blah language level not set”?

If so, that probably means Boolector’s build system isn’t passing in `--3` when calling Cython (and it should if we’re building for Py3).

(sorry for the vagueness; not at a computer)